full/build.gradle

```diff
@@ -18,17 +16,24 @@ spotless {
     }
 }
 
-archivesBaseName = "apoc"
-
+base {
+    archivesName = "apoc"
+}
 
 jar {
     manifest {
-        attributes 'Implementation-Version': version
+        attributes 'Implementation-Version': archiveVersion
     }
+    archiveClassifier = 'full'
```

Comment:
Make sure this is not breaking the jar classifiers like https://github.com/neo4j/apoc/issues/259#issuecomment-1403779626